src/commands/export-sql.ts

```diff
@@ -290,36 +336,34 @@ export class ExportSQLCommand {
 	/**
 	 * Checks if the export job's preflight step is successful
 	 *
-	 * @param {any} job The export job
-	 * @return {boolean} True if the preflight step is successful
+	 * @param job The export job
+	 * @return True if the preflight step is successful
 	 */
-	isPrepared( job ) {
-		const step = job?.progress.steps.find( st => st.id === 'preflight' );
+	private isPrepared( job: Job | undefined ): boolean {
+		const step = job?.progress?.steps?.find( st => st?.id === 'preflight' );
```

Comment:
All these optional accesses should be safe as customers hasn't come back to us with an issue on these commands...

Reply:
We have a test like

```js
		it( 'should return true if the preflight step has completed', async () => {
			const isPrepared = await exportCommand.isPrepared( job );
			expect( isPrepared ).toEqual( true );
		} );
```

If/when we rewrite tests to TypeScript, they will fail :-(

Reply:
Oh no! All right, I'll `unprivate` these. I'll also refactor the tests to typescript

Thanks for the review!

PS: I usually do `exportCommand['isPrepared']` and make prettier and eslint ignore autoformatting on it but in the spirit of changing less implementation, I'll unprivate it.